lib/elixir/lib/range.ex

```diff
@@ -21,6 +21,22 @@ defmodule Range do
       iex> last
       3
 
+  A Range includes members from Enum such as reduce, count and member?
```

Comment:
@lexmag Yes, I am referring to the implementation of the Enumerable protocol, thank you for noting for me the distinction between that and Ranges as inputs of Enum functions. I think it would be good to clarify that for all modules that provide an implementation of the `Enumerable` protocol so that it's clear when looking at `h Range` or `h Map` to learn about functionality that a module contains. 

I could take as a "to do" to add this detail in moduledocs for `Range`, `Map` and `List`